pages/index.jsx

```diff
@@ -17,6 +18,7 @@ var Index = React.createClass({
           <script src="https://js.stripe.com/v1/"></script>
           <script src="/js/stripe.js"></script>
           <script src="/js/ga.js"></script>
+          <Pontoon/>
```

Comment:
What's the downside to just adding `<script src='https://pontoon.mozilla.org/pontoon.js'></script>` always?